### PR TITLE
chore!: remove unused endpoints + FHIR + Elastic

### PR DESCRIPTION
**Pull request by davidlougheed**

### CODE REVIEW

**Comment by davidlougheed** [comment]
@gsfk could you verify i'm not removing anything you use here?

**Review by gsfk — APPROVED** [review]

File: chord_metadata_service/patients/models.py [hunk 1]
```diff
@@ -71,10 +71,6 @@ def get_project_id(self) -> str | None:
     gender = JSONField(blank=True, null=True, validators=[ontology_validator],
                        help_text='Self-identified gender')
 
-    # FHIR specific
-    active = models.BooleanField(default=False, help_text='Whether this patient\'s record is in active use.')
-    deceased = models.BooleanField(default=False, help_text='Indicates if the individual is deceased or not.')
-
```
Comment (v-rocheleau) [inline chord_metadata_service/patients/models.py:77]: These fields are still in the Individual schema and descriptions dict.

**Review by v-rocheleau — CHANGES_REQUESTED** [review]
Small request, will take a closer look at interpretation changes first thing tomorrow.

File: chord_metadata_service/patients/models.py (hunk 1 above)
Comment (davidlougheed) [reply chord_metadata_service/patients/models.py:77]: good point

**Review by v-rocheleau — CHANGES_REQUESTED** [review]
Looks good, validated that the endpoints removed are not used elsewhere.
Few things:
- There is still the (now unused) `fhir_utils.py` file, which links to hl7 and FHIR mapping files.
- `restapi/schemas` has an unused `FHIR_BUNDLE_SCHEMA`
- `restapi/semantic_mappings/README.md` can probably be removed

**Review by v-rocheleau — COMMENTED** [review]
Lint fails, otherwise LGTM!

**Pull request merged**